recipes/polars/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -ex
+
+# todo: Check +avx when supported in conda-forge
+export RUSTFLAGS='-C target-feature=+fxsr,+sse,+sse2,+sse3,+ssse3,+sse4.1,+sse4.2,+popcnt,+fma'
```

Comment:
Remove this so that the default options are used and when conda-forge moves forward, it'll be done automatically.

Reply:
Note that we use these feature flags in polars. Or are they moved somewhere else? I am not really familiar with conda feedstocks.

Reply:
Yes thats correct but these are reasonable defaults that practically all machines have. The rust defaults are super conservative because they also ship to embedded.

 Note that we don't activate avx. That one is not yet broadly supported.

Reply:
Would you be okay going ahead with the defaults and looking at the flags in a separate pr? I can open a new one after this for that purpose. @isuruf ?